### PR TITLE
Config API reference for v1.25

### PR DESCRIPTION
**Pull request by tengqm**
This PR batch updates the references for configuration APIs, generated using the [genref tool](kubernetes-sigs/reference-docs), based on the new release tags for kubernetes v1.25.

### CODE REVIEW

File: content/en/docs/reference/config-api/kubeadm-config.v1beta2.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ configuration types to be used during a <code>kubeadm init</code> run.</p>
 </span><span style="color:#bbb"></span><span style="color:#000;font-weight:bold">etcd</span>:<span style="color:#bbb">
 </span><span style="color:#bbb">  </span><span style="color:#998;font-style:italic"># one of local or external</span><span style="color:#bbb">
 </span><span style="color:#bbb">  </span><span style="color:#000;font-weight:bold">local</span>:<span style="color:#bbb">
-</span><span style="color:#bbb">    </span><span style="color:#000;font-weight:bold">imageRepository</span>:<span style="color:#bbb"> </span><span style="color:#d14">&#34;registry.k8s.io&#34;</span><span style="color:#bbb">
+</span><span style="color:#bbb">    </span><span style="color:#000;font-weight:bold">imageRepository</span>:<span style="color:#bbb"> </span><span style="color:#d14">&#34;k8s.gcr.io&#34;</span><span style="color:#bbb">
```
Comment (jihoon-seo) [inline content/en/docs/reference/config-api/kubeadm-config.v1beta2.md:162]: IMO the domain of container registry should be `registry.k8s.io`, not `k8s.gcr.io`; please see #35803.

File: content/en/docs/reference/config-api/kubelet-config.v1beta1.md [hunk 2]
```diff
@@ -943,7 +943,7 @@ Default: &quot;&quot;</p>
 <td>
    <p>systemReservedCgroup helps the kubelet identify absolute name of top level CGroup used
 to enforce <code>systemReserved</code> compute resource reservation for OS system daemons.
-Refer to <a href="https://git.k8s.io/design-proposals-archive/node/node-allocatable.md">Node Allocatable</a>
+Refer to <a href="https://git.k8s.io/community/contributors/design-proposals/node/node-allocatable.md">Node Allocatable</a>
```
Comment (jihoon-seo) [inline content/en/docs/reference/config-api/kubelet-config.v1beta1.md:946]: The target page of this link is 404 Not Found.

**Review by jihoon-seo — COMMENTED** [review]
@tengqm PTAL!

File: content/en/docs/reference/config-api/kubelet-config.v1beta1.md (hunk 2 above)
Comment (tengqm) [reply content/en/docs/reference/config-api/kubelet-config.v1beta1.md:946]: Please propose a fix in the upstream source [https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubelet/config/v1beta1/types.go#L628].

File: content/en/docs/reference/config-api/kubeadm-config.v1beta2.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/reference/config-api/kubeadm-config.v1beta2.md:162]: Okay, this needs a resync. Will work on it.

File: content/en/docs/reference/config-api/kubelet-config.v1beta1.md (hunk 2 above)
Comment (jihoon-seo) [reply content/en/docs/reference/config-api/kubelet-config.v1beta1.md:946]: I opened a PR: https://github.com/kubernetes/kubernetes/pull/111997 .

**Comment by sftim** [comment]
@tengqm please set a PR description - it won't be obvious to everyone what this is about.

**Comment by tengqm** [comment]
> @tengqm please set a PR description - it won't be obvious to everyone what this is about.

Done.

File: content/en/docs/reference/config-api/kubeadm-config.v1beta2.md (hunk 1 above)
Comment (reylejano) [reply content/en/docs/reference/config-api/kubeadm-config.v1beta2.md]: has to be changed upstream

**Comment by reylejano** [comment]
Preview:
https://deploy-preview-36208--kubernetes-io-main-staging.netlify.app/docs/reference/config-api/

**Pull request merged**